source/Willow.package/Html5ComponentSupplier.class/instance/checkboxLabeled.onModel.offModel.applying..st

```diff
@@ -0,0 +1,8 @@
+Supplying
+checkboxLabeled: aLabel onModel: anObjectToUseWhenOn offModel: anObjectToUseWhenOff applying: aCommandToComponent
```

Comment:
I'm using aRenderable instead of aLabel to make it clear that any renderable would work. aLabel gives more the idea that it has to be a text.